indolaw-nextjs/utils/theme.tsx

```diff
@@ -9,6 +9,7 @@ export const colors = {
   text: "#514d48",
   textHover: '#818181',
   linkText: "#373d3f",
+  subcontent: "#d3d3d3",
```

Comment:
mmmm this is very small nitpick, but this is for the background of the penjelasan box right?

so far, for background colors, it's always named `background`; so background of the law is `colors.background`, background for the tray is `colors.tray.background`, maybe this can be `colors.subcontent.background`?